src/store/apps/index.ts

```diff
@@ -1,8 +1,4 @@
-import {
-  createSlice,
-  createAction,
-  PayloadAction,
-} from '@reduxjs/toolkit';
+import { createSlice, createAction, PayloadAction } from '@reduxjs/toolkit';
```

Comment:
as with arrays, i find import statements are easier to manage if each element is on a separate line.

Reply:
prettier doesn't have a solution for it, we should find a plugin for it

Reply:
no. let's just do the minimum with this PR, and get it mergted. this stuff is all auto-fixable, so let's just add/adjust the rules as we go.